pages/common/jhsdb.md

```diff
@@ -0,0 +1,19 @@
+# jhsdb
+
+> Attach to a Java process or launch a postmortem debugger to analyze the content of a core dump from a crashed Java Virtual Machine (JVM).
+
+- Using pid to connect to a hung process:
+
+`jhsdb jstack --pid {{pid}}`
+
+- Open a core dump for analysis:
+
+`jhsdb clhsdb --core {{./core}} --exe {{/path/to/jdk/bin/java}}`
+
+- Start a debug server:
+
+`jhsdb debugd --pid {{pid}} --serverid {{server_id}}`
```

Comment:
How to determine a server id? Is it a hostname or an address?

Reply:
Docs say: An optional unique ID to use when multiple debug servers are running on the same remote host. Changing to {{optional_unique_id}}.